### PR TITLE
Fix tag filtering

### PR DESCRIPTION
**Pull request by todi1856**
Bug fix for https://jira.unity3d.com/browse/ALB-38

Fixed tag filtering, instead of using logic "incoming_tag contains specified_tag", we'll use "incoming_tag equals specified_tag", this allows to efficiently filter messages with tags like Unity, UnityPlugin. Because previously if you specify to filter with Unity, UnityPlugin tag would be picked up too.

Added automated test to cover this scenario.

### CODE REVIEW

File: com.unity.mobile.android-logcat/Editor/AndroidLogcat.cs [hunk 1]
```diff
@@ -354,7 +354,7 @@ private bool MatchTagsFilter(string tagInMsg)
         {
             foreach (var tag in Tags)
             {
-                if (tagInMsg.Contains(tag))
+                if (tagInMsg.Equals(tag))
```
Comment (aurimasc) [inline com.unity.mobile.android-logcat/Editor/AndroidLogcat.cs:357]: Wouldn't Tags.Contains(tagInMsg) return the same?

**Review by aurimasc — APPROVED** [review]
Code improvement suggested.

File: com.unity.mobile.android-logcat/Editor/AndroidLogcat.cs (hunk 1 above)
Comment (todi1856) [reply com.unity.mobile.android-logcat/Editor/AndroidLogcat.cs]: Nice catch - https://github.com/Unity-Technologies/com.unity.mobile.logcat/pull/184/commits/6a3174f1a11eeeabec828fb9a1c5c158d748fa9a

**Review by IGuscin — APPROVED** [review]
Tested that tag filtering now works properly, matching only exact tags

**Pull request merged**